modin/pandas/base.py

```diff
@@ -139,21 +134,25 @@ def _add_sibling(self, sibling):
 
     def _build_repr_df(self, num_rows, num_cols):
         """
-        Implement [METHOD_NAME].
-
-        TODO: Add more details for this docstring template.
+        Build pandas DataFrame for string representation.
 
         Parameters
         ----------
-        What arguments does this function have.
-        [
-        PARAMETER_NAME: PARAMETERS TYPES
-            Description.
-        ]
+        num_rows : int
+            Number of rows to show in string representation. If number of
+            rows in this dataset is greater than `num_rows` then half of
+            `num_rows` rows from the beginning and half of `num_rows` rows
+            from the end are shown.
+        num_cols : int
+            Number of columns to show in string representation. If number of
+            columns in this dataset is greater than `num_cols` then half of
+            `num_cols` columns from the beginning and half of `num_cols`
+            columns from the end are shown.
 
         Returns
         -------
-        What this returns (if anything)
+        `pandas.DataFrame` or `pandas.Series`
+        pandas dataset with `num_rows` or fewer rows and `num_cols` or fewer columns.
```

Comment:
```suggestion
        pandas.DataFrame or pandas.Series
            pandas dataset with `num_rows` or fewer rows and `num_cols` or fewer columns.
```

Reply:
There is a problem with `doc_checker.py` now. For low case `pandas` I get an error
```
ERROR:modin.pandas.base.BasePandasDataset._build_repr_df:135:RT04:Return value description should start with a capital letter
```
and for capital `Pandas` it gives error
```
ERROR:modin.pandas.base.BasePandasDataset._build_repr_df:135:MD02:Spelling error in line: 19, found: 'Pandas', reference: 'pandas'
```

Reply:
Just do not write the word pandas, this can be understood by the type that is located on the line above.

Reply:
@anmyachev I believe we can fix that in #3001, too.

Reply:
Ok thanks. `# noqa: MD02` helped to fix this warning.